src/components/input/input.js

```diff
@@ -480,6 +530,7 @@ function placeholderDirective($log) {
   }
 }
 
+var visibilityDirectives = ['ngIf', 'ngShow', 'ngHide', 'ngSwitchWhen', 'ngSwitchDefault'];
```

Comment:
please add the breakpoint attributes:

*  show, show-xs, show-gt-xs, etc.
*  hide, hide-xs, hide-gt-xs, etc.